src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4065,34 +4065,16 @@ internal void ImportCmdletsFromAssembly(Assembly assembly, PSModuleInfo module)
         ";
 
         /// <summary>
-        /// This is the default function to use for clear-host. On Windows it rewrites the
-        /// host, and on Linux, it delegates to the native binary, 'clear'.
+        /// This is the default function to use for clear-host.
         /// </summary>
         internal static string GetClearHostFunctionText()
         {
-            if (Platform.IsWindows)
-            {
-                return @"
-$RawUI = $Host.UI.RawUI
-$RawUI.CursorPosition = @{X=0;Y=0}
-$RawUI.SetBufferContents(
-    @{Top = -1; Bottom = -1; Right = -1; Left = -1},
-    @{Character = ' '; ForegroundColor = $rawui.ForegroundColor; BackgroundColor = $rawui.BackgroundColor})
-# .Link
-# https://go.microsoft.com/fwlink/?LinkID=225747
-# .ExternalHelp System.Management.Automation.dll-help.xml
-";
-            }
-            else
-            {
-                // Porting note: non-Windows platforms use `clear`
-                return @"
-& (Get-Command -CommandType Application clear | Select-Object -First 1).Definition
+            return @"
+[Console]::Clear()
 # .Link
```

Comment:
This isn't going to work for non-console hosts like ISE, VSCode(?) and remoting. On those platforms, you still need to use the `$RawUI` portability APIs. A nice solution would be to actually add a new `RawUI.Clear()` API and then implement that using `Console.Clear()` in the console host but that may end up being a bunch of work...